data-processing/requirements.txt

```diff
@@ -3,7 +3,7 @@ beautifulsoup4==4.8.0
 lxml==4.4.2
 nltk==3.4.5
 numpy==1.17.4
-opencv-python==3.4.8.29
+opencv-python-headless==3.4.8.29
```

Comment:
Ran into missing `libSM.so` error. Getting the `headless` version is the solution according to https://stackoverflow.com/a/56738577.